tensorflow/contrib/rnn/python/ops/core_rnn_cell_impl.py

```diff
@@ -83,12 +83,14 @@ def call(self, inputs, state):
 class GRUCell(RNNCell):
   """Gated Recurrent Unit cell (cf. http://arxiv.org/abs/1406.1078)."""
 
-  def __init__(self, num_units, input_size=None, activation=tanh, reuse=None):
+  def __init__(self, num_units, input_size=None, activation=tanh, reuse=None,
+               initializer=None):
     super(GRUCell, self).__init__(_reuse=reuse)
     if input_size is not None:
       logging.warn("%s: The input_size parameter is deprecated.", self)
     self._num_units = num_units
     self._activation = activation
+    self._initializer = initializer
```

Comment:
@lukaszkaiser 

Thanks for your reply.
I have one more question.

According to the keras documentation, it seems `kernel_initializer` is same with the `initializer` parameter I added.

```
kernel is a weights matrix created by the layer, and bias is a bias vector created by the layer (only applicable if use_bias is True).
```

If `initializer` and `kernel_initializer` are same, is it okay to add `bias_initializer` only?